rx_signal_processing/dsp.hpp

```diff
@@ -63,47 +79,81 @@ class DSPCore {
     void send_ack();
     void send_timing();
 
-
+//TODO(keith): May remove sizes as member variables.
```

Comment:
Given how this design turned out, they are no longer useful as class members. They are only used in one place. I thought I might need them elsewhere, but seems to not be the case.